third_party/terraform/utils/provider.go.erb

```diff
@@ -181,6 +181,7 @@ func Provider() terraform.ResourceProvider {
 			"google_container_engine_versions":                 dataSourceGoogleContainerEngineVersions(),
 			"google_container_registry_image":                  dataSourceGoogleContainerImage(),
 			"google_container_registry_repository":             dataSourceGoogleContainerRepo(),
+			"google_dns_key":                                   dataSourceDNSKey(),
```

Comment:
I think it makes sense for this resource to be named `google_dns_keys` instead. It returns a collection of keys rather than an individual key

Reply:
Got it!